app/controllers/nominations_controller.rb

```diff
@@ -1,7 +1,45 @@
 # frozen_string_literal: true
 
 class NominationsController < ApplicationController
-  before_action :load_nomination_request_form, except: %i[choose_location]
+  before_action :load_nomination_request_form, except: %i[choose_location index resend_email_after_link_expired link_expired already_nominated]
+
+  def index
+    @nomination_email = NominationEmail.find_by(token: params[:token])
+    @token = params[:token]
+
+    if @nomination_email.nil?
+      redirect_to link_invalid_nominations_path
```

Comment:
I don't think this has been covered in any of the tests

Reply:
There wasn't any designs for that scenario so just mocked the functionality until we will have screens for that